src/Microsoft.Data/PrimitiveColumn.cs

```diff
@@ -181,6 +182,32 @@ protected override void SetValue(long rowIndex, object value)
             }
         }
 
+        public override double Median()
+        {
+            // Not the most efficient implementation. Using a selection algorithm here would be O(n) instead of O(nLogn)
```

Comment:
I'm not very inclined to write an efficient selection method. Primarily because I want efficient stats to be provided by an external lib. LinqStatistics for example